content/en/blog/_posts/2022-05-03-kubernetes-release-1.24.md

```diff
@@ -0,0 +1,238 @@
+* [Beta APIs are off by default](https://github.com/kubernetes/enhancements/issues/2784): Newly added beta APIs are not enabled by default.
+
+### Major Changes
+
+This release saw two major changes: 
+
+* [Dockershim Removal](https://github.com/kubernetes/enhancements/issues/2221)
+* [Beta APIs are off by Default](https://github.com/kubernetes/enhancements/issues/3136)
+
+### Release Notes
+
+Check out the full details of the Kubernetes 1.24 release in our [release notes](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.24.md).
+    
+### Availability
+
+Kubernetes 1.24 is available for download on [GitHub](https://github.com/kubernetes/kubernetes/releases/tag/v1.24.0).
+To get started with Kubernetes, check out these [interactive tutorials](/docs/tutorials/) or run local 
+Kubernetes clusters using containers as “nodes”, with [kind](https://kind.sigs.k8s.io/).
+You can also easily install 1.24 using [kubeadm](/docs/setup/independent/create-cluster-kubeadm/).
+
+### Release Team
+
+This release would not have been possible without the combined efforts of committed individuals 
+comprising the Kubernetes 1.24 release team. This team came together to deliver all of the components 
+that go into each Kubernetes release, including code, documentation, release notes, and more. 
+
+Special thanks to James Laverack, our release lead, for guiding us through a successful release cycle, 
+and to all of the members of the release team for the time and effort they put in to deliver the v1.24 
+release for the Kubernetes community. 
+
+### Release Theme and Logo 
+
+**Kubernetes 1.24: Stargazer**
+
+{{< figure src="/images/blog/2022-05-03-kubernetes-release-1.24/kubernetes-1.24.png" alt="" class="release-logo" >}}
+
+The theme for Kubernetes 1.24 is _Stargazer_.
+
+Generations of people have looked to the stars in awe and wonder, from ancient astronomers to the 
+scientists who built the James Webb Space Telescope. The stars have inspired us, set our imagination 
+alight, and guided us through long nights on difficult seas.
+
+With this release we gaze upwards, to what is possible when our community comes together. Kubernetes 
+is the work of hundreds of contributors across the globe, and thousands of end-users supporting 
+applications that serve millions. Every one is a star in our sky, helping us chart our course.
+
+The release logo is made by [Britnee Laverack](https://www.instagram.com/artsyfie/), and depicts a telescope set upon starry skies and the 
+[Pleiades](https://en.wikipedia.org/wiki/Pleiades), often known in mythology as the “Seven Sisters”. The number seven is especially auspicious 
+for the Kubernetes project, and is a reference back to our original “Project Seven” name.
+
+This release of Kubernetes is named for those that would look towards the night sky and wonder — for 
+all the stargazers out there. :sparkles:
```

Comment:
I don't think the emoji name renders in the actual preview, so it'll have to be the emoji itself.
```suggestion
all the stargazers out there. ✨
```